ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -206,59 +212,51 @@ export class AptosClient {
     sender: MaybeHexString,
     payload: Gen.TransactionPayload,
```

Comment:
yeah. Initially, when I started the PR, I didn't want to break the signature of the original functions. But since this is a breaking change in the end, yeah, why not?